content/house/_index.md

```diff
@@ -4,5 +4,27 @@ layout: leftalign
 menu: 
   main:
     weight: 1
+featured_image: 'static/images/haus2.jpeg'
 ---
-Das im traditionellen Stil restaurierte, rustikale Olivenbauernhäuschen hat seine toskana-typische Atmosphäre bewahrt und besitzt dennoch modernen Komfort (Wasch-und Spülmasche, Satelliten-TV, Espressomaschine, Insektenschutzrollos). Es verfügt auf zwei Wohnebenen über ein Wohnzimmer (Studiocharakter), eine Wohnküche mit Kaminofen, zwei Schlafzimmer, Bad (Duschkabine) und ein WC. Die im Haus befindliche Treppe kann in beide Richtungen mit Treppenschutzrollos für Kinder gesichert werden. Alle Fenster und Türen sind vergittert. 
\ No newline at end of file
+Das im traditionellen Stil restaurierte, rustikale Olivenbauernhäuschen hat seine toskana-typische Atmosphäre bewahrt und besitzt dennoch modernen Komfort (Wasch-und Spülmasche, Satelliten-TV, Espressomaschine, Insektenschutzrollos). Es verfügt auf zwei Wohnebenen über ein Wohnzimmer (Studiocharakter), eine Wohnküche mit Kaminofen, zwei Schlafzimmer, Bad (Duschkabine) und ein WC. Die im Haus befindliche Treppe kann in beide Richtungen mit Treppenschutzrollos für Kinder gesichert werden. Alle Fenster und Türen sind vergittert. 
+
+---
+---
+
+Das Schlafzimmer: 
+
+!['Schlafzimmer1'](/static/images/bett.jpeg) !['Schlafzimmer2'](/static/images/bett2.jpeg)
```

Comment:
```suggestion
!['Das Bett'](/static/images/bett.jpeg) !['Das Bett'](/static/images/bett2.jpeg)
```

Die in [] angegebenen Texte sind wichtig für Screenreader (für Sehgeschädigte) und Suchmaschinen. Hier bitte sinnvollen Text eingeben.